reports/mobile_clients/mobile-clients.ipynb

```diff
@@ -73,11 +73,15 @@
     "        except:\n",
     "            profileDate = None\n",
     "\n",
-    "    # Create date should already be in ISO format\n",
-    "    creationDate = ping[\"created\"]\n",
-    "    if creationDate is not None:\n",
-    "        # This is only accurate because we know the creation date is always in 'Z' (zulu) time.\n",
-    "        creationDate = dt.datetime.strptime(ping[\"created\"], \"%Y-%m-%d\")\n",
+    "    # Create date can be an improper string (~.03% of the time, so ignore)\n",
+    "    try: \n",
+    "        # Create date should already be in ISO format\n",
+    "        creationDate = ping[\"created\"]\n",
+    "        if creationDate is not None:\n",
+    "            # This is only accurate because we know the creation date is always in 'Z' (zulu) time.\n",
+    "            creationDate = dt.datetime.strptime(ping[\"created\"], \"%Y-%m-%d\")\n",
+    "    except Exception:\n",
```

Comment:
It would be better to specify the exact exception you want to catch. It should be ValueError iirc.